includes/class-wcmp-admin.php

```diff
@@ -100,11 +100,6 @@ public function order_list_return_shipment_options($order, $hide = true) {
         if ($shipping_country != 'NL' && ! WooCommerce_MyParcel()->export->is_eu_country($shipping_country)) {
             return;
         }
-        $order_id = WCX_Order::get_id($order);
-        $shipment_options = WooCommerce_MyParcel()->export->get_options($order);
-        $myparcel_options_extra = WCX_Order::get_meta($order, '_myparcel_shipment_options_extra');
-        $package_types = WooCommerce_MyParcel()->export->get_package_types('return');
-        $recipient = WooCommerce_MyParcel()->export->get_recipient($order);
```

Comment:
Zijn deze dingen niet nodig in wcmp-order-return-shipment-options.php ?

Reply:
Werden niet gebruikt :/